src/components/login/SSOLogin.tsx

```diff
@@ -538,7 +538,7 @@ export default class SSOLogin extends Component<SSOLoginProps, SSOLoginState> {
                         <input
                             type="text"
                             className="form__input"
-                            value={this.state.ssoConfig.url}
+                            value={this.state.ssoConfig.url.length === 0 ? `${window.location.origin}/orchestrator` : this.state.ssoConfig.url}
```

Comment:
We can write this as ={this.state.ssoConfig.url || `${window.location.origin}/${process.env.REACT_APP_ORCHESTRATOR_ROOT}`}